src/telegram2discord/setup.ts

```diff
@@ -152,9 +153,9 @@ export function setup(
 			tgBot.catch((err: any) => {
 				// The docs says timeout errors should always be rethrown
 				// @ts-ignore TODO: Telefraf does not exprt the TimoutError, alternative implementation needed
-				if (err instanceof telegraf.TimeoutError) {
-					throw err;
-				}
+				// if (err instanceof telegraf.TimeoutError) {
+				// 	throw err;
+				// }
 
```

Comment:
What's the reason for this

Reply:
bot crashes when this error actually occurs

Reply:
Well, the comment says it should be like that. So the root problem has to be somewhere else

Reply:
Yes. It crashes on any Error, because no access to telegraf.TimeoutError.
If you think that it's the right way to kill the bot on any Tg error - maybe just leave one row with "throw err" than ?

Reply:
oh, let's remove that commented stuff completely then